### ISSUE TITLE
[ANDROID] Sometimes crash on onActivityResult(app store log)

### ISSUE THREAD

**Issue by ekleop** [j3k0/cordova-plugin-purchase#685]
```
Cordova 8.0.0, cordova-android 7.0.0
Device: Many, from Android 5 to 8
Plugin Version: 7.1.0
```

Have this error 3-5 times a day.

```
Samsung Galaxy J7(2016) (j7xelte), 2048MB RAM, Android 6.0
Report 4 of 4

java.lang.RuntimeException: 
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4157)
  at android.app.ActivityThread.handleResumeActivity (ActivityThread.java:4251)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3362)
  at android.app.ActivityThread.access$1100 (ActivityThread.java:222)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1796)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:158)
  at android.app.ActivityThread.main (ActivityThread.java:7230)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1230)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1120)
Caused by: java.lang.RuntimeException: 
  at android.app.ActivityThread.deliverResults (ActivityThread.java:4926)
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4143)
Caused by: java.lang.NullPointerException: 
  at com.smartmobilesoftware.inappbilling.InAppBillingPlugin.onActivityResult (InAppBillingPlugin.java:453)
  at org.apache.cordova.CordovaInterfaceImpl.onActivityResult (CordovaInterfaceImpl.java:159)
  at org.apache.cordova.CordovaActivity.onActivityResult (CordovaActivity.java:359)
  at android.app.Activity.dispatchActivityResult (Activity.java:7138)
  at android.app.ActivityThread.deliverResults (ActivityThread.java:4922)
```
Would be glad for help

**Comment by aabdolla** [comment]
Did you ever solve this?

**Comment by creakosta** [comment]
I have same error in log

**Comment by Dexus** [comment]
This should be fixed with #713
